homework/week1/RosmaryFC/HoroscopeHW20150508/app/src/main/res/layout/fragment_signs.xml

```diff
@@ -0,0 +1,31 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
could include some comments as to what this specific fragment does. Implement comments by doing <!--- to open comment and --> to close